### ISSUE TITLE
Duplicating Projects / Making custom Project Templates

### ISSUE THREAD

**Issue by unfa** [go-gitea/gitea#18943]
### Feature Description

I would like to use Gitea Project to manage sprints in a team.

The issue is - I don't see a way to create a custom Project Template or even duplicate Projects so I can reuse a board layout for managing issues. There's two templates (Basic Kanban and Bug Triage) but I don't see a way to add more.

Is there no such feature, or I am just missing something?
I am using Codeberg.

---

Another thing is - maybe I am misunderstanding the intended workflow?
Maybe the Project is supposed to be reused forever, and the milestones would define sprints?

Either way, if I want to create 4 projects with (nearly) identical board layouts it'd be really wasteful to create them manually, so at least an option to duplicate a Project seems necessary.

---
Thank you for your great work!

### Screenshots

![image](https://user-images.githubusercontent.com/491704/155990420-268d38ef-0381-4926-aa84-bf7e4d492943.png)

**Comment by davipro34** [comment]
Hi,
I pray for such functionality.

**Comment by earl-designs** [comment]
Yes please I would also like to use Gitea to manage Projects and not have to use another tool

**Comment by JanRomero** [comment]
It appears that for now you can only have the two stock project types, but you can reconfigure their boards in `/custom/conf/app.ini`:

```INI
[project]
PROJECT_BOARD_BASIC_KANBAN_TYPE = Hirschcarpaccio, Preiselbeersauce
PROJECT_BOARD_BUG_TRIAGE_TYPE = Schnitzel, Pommes, Salat
```

Not sure if you can do colours, too…

**Comment by austin-berry** [comment]
This would be an amazing feature!

Having something similar to how GitHub has the Gantt chart for workflows and even tables. Project management would be a huge feature for Gitea!

I'd love to help implement such a feature

**Comment by austin-berry** [comment]
That and designing custom templates!

**Comment by unfa** [comment]
> It appears that for now you can only have the two stock project types, but you can reconfigure their boards in `/custom/conf/app.ini`

Well, it is an option for people self-hosting Gitea, but what about public instances like Codeberg?

**Comment by Snuupy** [comment]
I see github projects has table views (including group by, sort by, slice by) to organize issues in the table. I find this very useful for large backlogs.

**Comment by AliMickey** [comment]
Has there been any progress on this that I have missed?

Being able to create custom project templates would be very helpful, a custom set of boards with predefined tasks within each for example.

**Comment by digitalextremist** [comment]
Seems the status of this functionality is still where it stood in `2022` but is all-the-more necessary now!

**Comment by checkmeck** [comment]
I would also appreciate such a feature:
- Customizing project templates via the UI
- Set the colors of the boards
- Have more than 2 templates / Create additional templates via the UI
- (Setting the default template for new projects)
